src/ensembl/production/metadata/models.py

```diff
@@ -72,7 +72,7 @@ class Attribute(Base):
     name = Column(String(128), nullable=False)
     label = Column(String(128), nullable=False)
     description = Column(String(255))
-    type = Column(String(32), nullable=False)
+    type = Column(Enum('string', 'percent', 'float','integer'), server_default=text("'string'"))
```

Comment:
```suggestion
    type = Column(Enum('string', 'percent', 'float','integer','bp'), server_default=text("'string'"))
```
please add 'bp' for base pair. 
We might come back to it another time with a new definition for that field (used for grouping, then we'll rename this one to `unit`).